hana/package.json

```diff
@@ -43,7 +43,8 @@
       "kinds": {
         "sql": {
           "[production]": {
-            "kind": "hana"
+            "kind": "hana",
+            "sslUseDefaultTrustStore": false
```

Comment:
why do we need that?:)

Reply:
I am pretty sure that this property would be required to be part of the `credentials` object. As that is what being send to the database driver. Also this seems to only be a problem with `hdb` it shouldn't be a problem with `@sap/hana-client`. Which would mean that it should probably go into:

https://github.com/cap-js/cds-dbs/blob/732a2f385074f50b87ff9715b8bdf48d28a36309/hana/lib/drivers/hdb.js#L23-L41

As the change was made in `hdb` to resolve a long running issue. I am hesitant to have our implementation overwrite the default. The main problem with this change is when the `HANAService` has a large connection pool or a lot of tenants. As out default `max` pool size is `10`. It would require ~1000 tenants to reach the reported state in the issue.

Additionally it is possible to use the previous `hdb` version which does not have this behavior to achieve the same state as this PR is trying to achieve. Which only becomes a problem when root issue shows up as it would require the `hdb` upgrade, but then having it disabled by `@cap-js/hana` would not allow CAP applications to connect to their HANA system at all anymore.

Reply:
Agreed that this is the better place. I moved it to `hdb.js` now.